### PR TITLE
Acquirer details

### PR DESCRIPTION
**Pull request by twarberg**
Screenshot: https://www.dropbox.com/s/etxznzb7zf1cwer/Screenshot%202016-01-27%2011.02.24.png?dl=0

### CODE REVIEW

File: appendixes/acquirer-details/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+title: Acquirer details
+---
+
+# Acquirer details
+
+## <a id="klarna"></a> Klarna
+
+### Requirements
+
+- Payment must contain full customer invoice address and/or shipping address. [API](/api/services/?scope=merchant#POST-payments---format-), [Form](/payments/form/)
```
Comment (ta) [inline appendixes/acquirer-details/index.md]: It is not clear to my why the links to Form and API are here? Perhaps add a few words or remove them?

File: appendixes/acquirer-details/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    </tr>
+    <tr>
+        <td>Netherlands</td>
+        <td>EUR</td>
+    </tr>
+  </tbody>
+</table>
+
+
+### Error codes
```
Comment (ta) [inline appendixes/acquirer-details/index.md]: I think we should keep information about errors in one page - http://tech.quickpay.net/appendixes/errors

Here we could tell where to find information about errors for external parties and link to docs if possible.

File: appendixes/acquirer-details/index.md (hunk 1 above)
Comment (twarberg) [inline appendixes/acquirer-details/index.md]: Removed

File: appendixes/acquirer-details/index.md (hunk 2 above)
Comment (twarberg) [inline appendixes/acquirer-details/index.md]: Moved to errors

**Pull request merged**